roles/pulp-webserver/defaults/main.yml

```diff
@@ -5,3 +5,5 @@ pulp_content_port: 24816
 pulp_api_host: 127.0.0.1
 pulp_api_port: 24817
 pulp_configure_firewall: auto
+pulp_webserver_static_dir: '/var/lib/pulp/assets/'
```

Comment:
1. Should be:
`pulp_webserver_static_dir: '{{ pulp_user_home }}/assets'`

2. We cannot assume that users will add a '/' at the end. Test that Apache functions without it.

3. Needs documentation in the role's README.md.

Reply:
Well, stripping the '/' is one way to solve this!